<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextBrowserDialog</class>
 <widget class="QWidget" name="TextBrowserDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRHT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QSpinBox" name="spin_bytes">
     <property name="maximum">
      <number>8192</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bytes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HexSpin" name="spin_offset"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="button_max_text">
     <property name="text">
      <string>Find Max Text Range</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexSpin</class>
   <extends>QSpinBox</extends>
   <header>GUI/hexspin.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spin_offset</tabstop>
  <tabstop>spin_bytes</tabstop>
  <tabstop>button_max_text</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
